documentation/api/query/v4/fact-contents.markdown

```diff
@@ -13,9 +13,9 @@ canonical: "/puppetdb/latest/api/query/v4/fact-contents.html"
 [factsets]: ./factsets.html
 [nodes]: ./nodes.html
 
-You can query fact information with greater power by using the '/fact-contents' endpoint. This endpoint provides the capability to descend into structured facts and query tree nodes deep within this data by using the concept of paths and values.
+You can query fact information with greater power by using the `/fact-contents` endpoint. This endpoint provides the capability to descend into structured facts and query tree nodes deep within this data by using the concept of paths and values.
 
-Structured fact data is normally represented as a hash, which allows hashes, arrays and real types as its values as you can see in this example:
+Structured fact data is normally represented as a hash, which allows hashes, arrays, and real types as its values:
```

Comment:
Structured fact data may be any valid JSON structure. Not sure what the best way to change this to reflect that is. The distinction is that the top-level structure of a structured fact may be either an array or a hash, so we should probably mention both.